### ISSUE TITLE
Create tests for APG design pattern example: Meter

### ISSUE THREAD

**Issue by IsaDC** [w3c/aria-at#570]
#### Applies To

* APG design pattern: [Meter](https://w3c.github.io/aria-practices/#meter)
* Specific example: [Meter](https://w3c.github.io/aria-practices/examples/meter/meter.html)
* Notes:
    1. Some screen readers treat meters as progress bars.  This behaviour is logically incorrect, and both the WAI-ARIA specification and the APG include text highlighting the fact that these two roles should not be used for the same purpose.  Accordingly, this test plan asserts the role of "meter", and screen readers that do not correctly identify that role will fail against the relevant assertions.
    2. Following on from point `#1`, many screen readers will automatically convey changes to a meter's value because that is the default behaviour for progress bars.  This test plan asserts that changes to the value of a meter should **not** be conveyed automatically, via speech or sound, for the following reasons:
        * meters and progress bars are not used for the same purpose (as outlined in point `#1`);
        * the value of a meter can change extremely frequently (e.g. if used to convey real time updates to network conditions); and
        * updates to a meter's value can take place over an indeterminate period of time, unlike a progress bar where the ETA of a process is often known or able to be estimated.
    3. In some browsers (e.g. Google Chrome), the value of a meter is rounded to less decimal  places than are actually included within `aria-valuenow`, meaning that the full `aria-valuenow` attribute value isn't available to screen reader users at all.  The WAI-ARIA specification does not indicate that such rounding should take place, nor does the Core Accessibility API Mappings documentation.  As such:
        * The full value of `aria-valuenow` is asserted in tests.
        * Authors wishing to provide a meaningfully rounded value for a particular application should utilise `aria-valuetext`, and a test case example using that method should be prepared in the future.

#### Testing Notes

This issue thread will house all documentation relating to the development of a test plan for the "Meter" APG example, which conforms to the "Meter" design pattern.  All follow-ups (including the test plan itself) will be added as new comments to facilitate notifications, but this initial comment will be updated with relevant details and URLs as needed for convenience.

#### Additional References

* [meter (role)](https://w3c.github.io/aria/#meter)
* [aria-labelledby (property)](https://w3c.github.io/aria/#aria-labelledby)
* [aria-valuenow (property)](https://w3c.github.io/aria/#aria-valuenow)
* [aria-valuemin (property)](https://w3c.github.io/aria/#aria-valuemin)
* [aria-valuemax (property)](https://w3c.github.io/aria/#aria-valuemax)
